conformance/requirements.txt

```
@@ -0,0 +1,6 @@
+tomli
+tomlkit
+pyright
+mypy
+pyre-check
+pytype
```

To what extent is it a goal or non-goal for this conformance test suite to be runnable on Windows? I know pytype can't be run on Windows; I think maybe pyre can't either? Obviously Windows devs won't be able to run the conformance tests for pyre and pytype locally, but it would be nice to be able to at least run the mypy/pyright tests locally on Windows.

Having pytype in the `requirements.txt` file unqualified means `pip install -r conformance/requirements.txt` fails with a very long and ugly error message on Windows. It's pretty easy to stop pip from trying to install pytype on Windows; you'd just need to do this (but you'd also need to make sure the script doesn't try to run the pytype tests if pytype isn't installed):

```suggestion
pyre-check
pytype; platform_system != "Windows"
```

If it's just not a goal for this conformance test suite to be runnable on Windows at all, it might be nice to add a note to that effect in the documentation

---

Thanks!